Eastern-Research-Group/lew
# Feature/update dependencies

While updating dependencies I noticed there were some that looked like they weren't being used (i.e., dayjs, minimist, and moment). I went ahead and removed these and everything still seems to work, but please verify these dependencies are not needed. 

## Related Issues:
* Closes #82 

## Main Changes:
* Updated node to 14.x
* Updated 3rd party dependencies
* Removed unnecessary dependencies (dayjs, minimist, and moment)

## Steps To Test:
1. Run `npm ci`
2. Look through the code and verify the dayjs, minimist, and moment dependencies are not needed
3. Run `npm run test`
4. Verify that all tests pass
5. Run `npm run start`
6. Verify the app works correctly, pay close attention to the date/time display
